Makefile

```diff
@@ -23,6 +26,7 @@ path_up_gateway: ## Run just the PATH gateway without any dependencies
 path_up_build_gateway: ## Run and build just the PATH gateway without any dependencies
 	docker compose up -d --build --no-deps path_gateway
 
+# TODO_UPNEXT(@adshmh): update path_up and path_down to use Tilt, and remove docker compose
 .PHONY: path_up
 path_up: ## Run the PATH gateway and all related dependencies
 	docker compose up -d
```

Comment:
Please add + modify this target:

```
.PHONY: config_shannon_localnet
config_shannon_localnet: ## Create a localnet config file for the Shannon relay
	cp localnet/path/config/shannon.example.yaml  localnet/path/config/.config.yaml
	@echo "Created ./localnet/path/config/.config.yaml"
	@echo "Please update the gateway_private_key value in the .config.yaml file with the correct value."
```

See my other comments.

Reply:
@adshmh @Olshansk Regarding all of the example/config files, I wanted to get your thoughts on consolidating all of them into a common directory?

The reason I suggest this is we are starting to get quite a number of example/template files around the repo and with the Envoy PRs there are even more of them.

Since they are primarily going to be used to run the PATH local Tilt stack, would it make sense to keep all of the example files (`shannon.example.yaml`, `morse.example.yaml`, etc.) all in a folder inside the `local` (or whatever we end up calling it) directory?

Reply:
@adshmh @Olshansk And further, regarding the Make target for copying the example files to the actual (git ignored) files, WDYT about a single script that copies all of them at once, rather than having many different make targets?

Reply:
> I wanted to get your thoughts on consolidating all of them into a common directory?

+1 Let's do it.

Create a `config/templates` directory and name things appropriately.

Reply:
> And further, regarding the Make targets for copying the example files to the actual (git ignored) files, WDYT about a single script that copies all of them at once, rather than having many different make targets?

Let's have make targets that do this one-by-one. 

These are the ones we'll reference in the docs so developers can follow along + understand.

You can add a "mega make target" that copies everything. These are the ones we'll as power-users + setting up CI/CD + testing. Just have this mega make target call the other baby make targets from within it.

Reply:
> > I wanted to get your thoughts on consolidating all of them into a common directory?
> 
> +1 Let's do it.
> 
> Create a `config/templates` directory and name things appropriately.

Not all of them are config related though. Many are envoy specific, which is why I was thinking of keeping everything under `local` since they are primarily going to used to run the localnet?